src/test/scala/is/hail/methods/LinearRegressionMultiPhenoSuite.scala

```diff
@@ -121,7 +121,7 @@ class LinearRegressionMultiPhenoSuite extends SparkSuite {
         "Sample",
         root = Some("sa.pheno"),
         config = TextTableConfiguration(types = Map("Pheno" -> TDouble), missing = "0"))
-      .linregMultiPheno(Array("sa.pheno.Pheno"), Array("sa.cov.Cov1", "sa.cov.Cov2 + 1 - 1"), "va.linreg", useDosages = true, 1, 0.0)
+      .linregMultiPheno(Array("sa.pheno.Pheno"), Array("sa.cov.Cov1", "sa.cov.Cov2"), useDosages = true)
```

Comment:
Yeah, because this is relying on the same `RegressionUtils.getPhenoCovCompleteSamples` as all the other regression methods, so it didn't seem necessary to test that here.